gridsome.server.js

```diff
@@ -22,23 +22,25 @@ function TailwindPlugin(api, options) {
   const purgecss = require('@fullhuman/postcss-purgecss')(purgeConfig)
 
   api.chainWebpack(config => {
-    config.module
-      .rule('css')
-      .oneOf('normal')
-      .use('postcss-loader')
-      .tap(options => {
-        shouldImport && options.plugins.push(postcssImport)
+    ['css', 'scss', 'sass', 'less', 'stylus'].forEach(lang => {
```

Comment:
Missing `pcss` and `postcss`.

Reply:
Yeah, I just added the `postcss` rule which also covers the `pcss` extension as you can see [here](https://github.com/gridsome/gridsome/blob/master/gridsome/lib/webpack/createBaseConfig.js#L151)

Reply:
You and I both at least, but people do use them. This only allows people that want to use them to use them together with Taiwlind. I don't see why this shouldn't be merged as long as that's the only reason not to. Clearly @amrnn thinks it's going to be useful.

I also find this useful since it enables support for `postcss` rules which makes things look right with my VSCode setup. I personally don't like calling `css` what's clearly not vanilla css yet.

If there's anything that could be done to help with this, just let me know.